focuspoints.lrdevplugin/DefaultPointRenderer.lua

```diff
@@ -128,6 +135,10 @@ function DefaultPointRenderer.getAutoFocusPoint(metaData)
     focusPointUsed = ExifUtils.findValue( metaData, DefaultPointRenderer.metaPrimaryAFPoint )
   end
 
+  if "(none)" == focusPointUsed then
+    focusPointUsed = -1
```

Comment:
nit: let's return nil here. focusPointUsed should be a reference and less of a number.